src/main/frontend/worker/db_worker.cljs

```diff
@@ -313,6 +242,20 @@
   (.exec db "PRAGMA locking_mode=exclusive")
   (.exec db "PRAGMA journal_mode=WAL"))
 
+(defn- gc-sqlite-dbs!
+  "Gc main db weekly and rtc ops db each time when opening it"
+  [sqlite-db client-ops-db datascript-conn {:keys [from-user?]}]
+  (let [last-gc-at (:kv/value (d/entity @datascript-conn :logseq.kv/graph-last-gc-at))]
```

Comment:
need to define `:logseq.kv/graph-last-gc-at` at ns `logseq.db.frontend.kv-entity`,
and it should be
- :rtc/ignore-entity-when-init-upload
- :rtc/ignore-entity-when-init-download

if we don't ignore it during upload/download-graph, it shouldn't actually affect the garbage collection